cutgeneratingfunctionology/igp/intervals.py

```diff
@@ -382,30 +416,33 @@ def scan_union_of_coho_intervals_minus_union_of_coho_intervals(interval_lists, r
     on = False
     for ((x, epsilon), delta, tag) in scan:
         was_on = on
-        if tag:                                       # interval event
+        if tag:  # interval event
             interval_indicator -= delta
-            assert(interval_indicator) >= 0, "interval_indicator should stay non-negative; required sort order must be violated"
-        else:                                           # remove event
+            assert (
+                       interval_indicator) >= 0, "interval_indicator should stay non-negative; required sort order must be violated"
```

Comment:
This is not a correct change. You are passing a tuple to `assert`, this is always true.

Reply:
I take this back.

Reply:
This formatting change with the proposed indentation is just misleading.

Reply:
Please just change it to 
```
assert interval_indicator >= 0, "interval_indicator should stay non-negative; required sort order must be violated"
```